scripts/vagrant/provision.sh

```diff
@@ -9,8 +9,6 @@ aws_access_key_id = not-a-real-id
 aws_secret_access_key = not-a-real-key' | tee ~/.boto
```

Comment:
Why are you using `tee` here? It seems like this can be handled with just a redirect.